.travis.yml

```diff
@@ -17,13 +17,21 @@ branches:
 services:
 - docker
 
+env:
+  global:
+    - CHEF_VERSION=15
+    - KITCHEN_YAML=kitchen.dokken.yml
+    - CHEF_LICENSE="accept-no-persist"
+
 before_install:
 - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
 - CHEF_LICENSE="accept-no-persist" sudo chef gem install webmock
 - eval "$(chef shell-init bash)"
 - chef --version
 - cookstyle --version
```

Comment:
chef --version should include all the version output you need. It includes cookstyle and kitchen and foodcritic is dead.